src/docfx/lib/markdown/validation/ContentValidationExtension.cs

```diff
@@ -144,5 +150,25 @@ static string GetContainerInlineContent(ContainerInline containerInline)
                 return content.ToString();
             }
         }
+
+        private class InclusionNode : ValidationNode, ICloneable
+        {
+            public string? IncludedFilePath { get; set; }
+
+            public InclusionNode()
+            {
+            }
+
+            protected InclusionNode(InclusionNode inclusionNode)
```

Comment:
where's this used?

Reply:
We need to clone node when inclusion node contains inclusion node. It is used in clone function.